File: packages/google_ml_kit_commons/ios/Classes/GoogleMlKitCommonsPlugin.m
#import "GoogleMlKitCommonsPlugin.h"

@implementation GoogleMlKitCommonsPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  FlutterMethodChannel* channel = [FlutterMethodChannel
      methodChannelWithName:@"google_ml_kit_commons"
            binaryMessenger:[registrar messenger]];
  GoogleMlKitCommonsPlugin* instance = [[GoogleMlKitCommonsPlugin alloc] init];
  [registrar addMethodCallDelegate:instance channel:channel];
}

- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
  result(FlutterMethodNotImplemented);
}

@end

File: packages/google_mlkit_commons/ios/google_mlkit_commons/Sources/google_mlkit_commons/GoogleMlKitCommonsPlugin.swift
import Flutter
import MLKitVision

@objc
public class GoogleMlKitCommonsPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let channel = FlutterMethodChannel(
      name: "google_mlkit_commons",
      binaryMessenger: registrar.messenger()
    )
    let instance = GoogleMlKitCommonsPlugin()
    registrar.addMethodCallDelegate(instance, channel: channel)
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    result(FlutterMethodNotImplemented)
  }
}
